### ISSUE TITLE
Repository Failure

### ISSUE THREAD

**Issue by posey311r** [sipcapture/homer-installer#78]
I am attempting to install Homer on clean CentOS 7 server and both the qxip_sipcapture and qxip_sipcapture-source repositories are failing. 

failure: repodata/repomd.xml from qxip_sipcapture: [Errno 256] No more mirrors to try.
https://packagecloud.io/qxip/sipcapture/el/7/x86_64/repodata/repomd.xml: [Errno 14] HTTPS Error 302 - Found
Generating yum cache for qxip_sipcapture-source...
https://packagecloud.io/qxip/sipcapture/el/7/SRPMS/repodata/repomd.xml: [Errno 14] HTTPS Error 302 - Found

**Comment by lmangani** [comment]
Hello @posey311r 
Which guide are you following exactly?

**Comment by posey311r** [comment]
I am using this page:https://github.com/sipcapture/homer-installer

After installing redhat-lsb-core, I copied and pasted the commands while root:cd /usr/src
wget https://github.com/sipcapture/homer-installer/raw/master/homer_installer.sh
chmod +x homer_installer.sh
./homer_installer.shDuring the install process, the repositories I listed fail.



    On Friday, February 12, 2021, 12:06:12 PM PST, Lorenzo Mangani <notifications@github.com> wrote:  
 
 


Hello @posey311r
Which guide are you following exactly?

**Comment by posey311r** [comment]
I have also attempted this using the manual install and quick install methods on this page:https://github.com/sipcapture/homer/wiki/Quick-Install

I have tried this on clean CentOS 7 and Debian 9 systems and the repositories fail each time.



    On Friday, February 12, 2021, 12:29:26 PM PST, Robert Posey <posey311r@yahoo.com> wrote:  
 
  I am using this page:https://github.com/sipcapture/homer-installer

After installing redhat-lsb-core, I copied and pasted the commands while root:cd /usr/src
wget https://github.com/sipcapture/homer-installer/raw/master/homer_installer.sh
chmod +x homer_installer.sh
./homer_installer.shDuring the install process, the repositories I listed fail.



    On Friday, February 12, 2021, 12:06:12 PM PST, Lorenzo Mangani <notifications@github.com> wrote:  
 
 


Hello @posey311r
Which guide are you following exactly?

**Comment by posey311r** [comment]
Any update on this?    On Friday, February 12, 2021, 12:32:37 PM PST, Robert Posey <posey311r@yahoo.com> wrote:  
 
  I have also attempted this using the manual install and quick install methods on this page:https://github.com/sipcapture/homer/wiki/Quick-Install

I have tried this on clean CentOS 7 and Debian 9 systems and the repositories fail each time.



    On Friday, February 12, 2021, 12:29:26 PM PST, Robert Posey <posey311r@yahoo.com> wrote:  
 
  I am using this page:https://github.com/sipcapture/homer-installer

After installing redhat-lsb-core, I copied and pasted the commands while root:cd /usr/src
wget https://github.com/sipcapture/homer-installer/raw/master/homer_installer.sh
chmod +x homer_installer.sh
./homer_installer.shDuring the install process, the repositories I listed fail.



    On Friday, February 12, 2021, 12:06:12 PM PST, Lorenzo Mangani <notifications@github.com> wrote:  
 
 


Hello @posey311r
Which guide are you following exactly?
